test/integ/quote-classic.test.ts

```diff
@@ -125,7 +135,9 @@ const checkPortionRecipientToken = (
     ? expectedPortionAmountReceived.subtract(actualPortionAmountReceived)
     : actualPortionAmountReceived.subtract(expectedPortionAmountReceived);
   // There will be a slight difference between expected and actual due to slippage during the hardhat fork swap.
-  const percentDiff = tokensDiff.asFraction.divide(expectedPortionAmountReceived.asFraction);
+  const percentDiff = tokensDiff.equalTo(ZERO)
+    ? new Percent(ZERO)
```

Comment:
nit: would prefer declaring ZERO as a constant and `BigNumber.from(0)` instead of importing from router sdk